packs/github/config.yaml

```diff
@@ -2,6 +2,7 @@
 token: ""
 user: ""
 password: ""
+BASE_URL: "https://github.com"  # can also be a GitHub Enterprise URL
```

Comment:
Please change to lowercase, `base_url`, and default it to `https://api.github.com`.